git-user-count/user-count.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+set -e
+
+basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+repositories=$basedir/repositories.txt
+userListFile="$basedir/gradle-enterprise-users.txt"
+userUniqueListFile="$basedir/gradle-enterprise-unique-users.txt"
```

Comment:
This is minor but it would be nice if the variable name could explicit if the file is an input or an output

Reply:
i think this is a good point. i added comments to clarify this in the script.